app/routes/datasets/versions.py

```diff
@@ -469,15 +472,14 @@ async def _get_raster_fields(asset: ORMAsset) -> List[RasterBandMetadata]:
     logger.debug(f"Processing data environment f{raster_data_environment}")
     for layer in raster_data_environment.layers:
         field_kwargs: Dict[str, Any] = {
-            "field_name": layer.name,
```

Comment:
from what I see `/<dataset>/<version>/fields` endpoint for rasters isn't being used internally or in GFW.  And, it's broken now with metadata changes and these updates get it to return valid response